Bang/GameEvents/CardEffects/DynamiteActionHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Bang.GameEvents.CardEffects.States;
+using Bang.Players;
+using Bang.PlayingCards;
+
+namespace Bang.GameEvents.CardEffects
+{
+    internal class DynamiteActionHandler : CardActionHandler
+    {
+        public override HandlerState ApplyEffect(Game.Gameplay gameplay, Player victim, BangGameCard card)
+        {
+            Logger.Info($"Player {gameplay.PlayerTurn.Character.Name} player dynamite card");
```

Comment:
Small typo:  _player dynamite card_. Should be _played_ or _plays_

Reply:
done